jetty-websocket/websocket-core-tests/pom.xml

```diff
@@ -94,7 +94,7 @@
   </build>
   <profiles>
     <profile>
-      <id>ci</id>
+      <id>autobahn</id>
```

Comment:
@olamy Will this not run `AutoBahnTests.java` by default or exclude the whole `websocket-core-tests` module?

Reply:
@lachlan-roberts  all tests will be run with this profile